### ISSUE TITLE
Imports not added when `(` starts at the next line

### ISSUE THREAD

**Issue by talw** [ujihisa/unite-haskellimport#16]
First of all, thanks for this awesome plugin!

If I may raise an issue I stumbled upon recently:

It seems that imports are not added to an import list where the opening
parenthesis `(` is written at the next line rather than the same line as the
import.

E.g.

Importing `formatTime` works in this case

```haskell
import Data.Time.Format (
  defaultTimeLocale, iso8601DateFormat)
```

```haskell
import Data.Time.Format (formatTime
  defaultTimeLocale, iso8601DateFormat)
```

But does not work in this case

```haskell
import Data.Time.Format
  (defaultTimeLocale, iso8601DateFormat)
```

The file does not change at all.

The reason I stumbled into this bug in the first place is that I started using
`hindent`, which transforms import lists that do not fit on one line to
something like the following:

```haskell
import Data.Time.Format
  ( defaultTimeLocale
  , formatTime
  , iso8601DateFormat
  , parseTimeM
  )
```

But after such a transformation takes place `unite-haskellimport` fails to add
any further imports.

It would be immensely appreciated if `unite-haskellimport` could support such a format, if at all
possible.

Thank you!

**Comment by ujihisa** [comment]
Yeah I know that that style is pretty common. `unite-haskellimport` should definitely support it too.

Like issue #15, I'll find my time to fix this after that. Or if anybody including you would like to try fixing, please try!

**Comment by talw** [comment]
No rush. I tried `brittany` code formatter and it works perfectly well with `unite-haskellimport`!
